tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/ESEmailAddressContactSearchEngine.java

```diff
@@ -45,7 +48,7 @@ public ESEmailAddressContactSearchEngine(ReactorElasticSearchClient client) {
     public Publisher<Void> index(AccountId accountId, EmailAddressContact contact) {
         return Mono.fromRunnable(() -> {
             String content = Try.ofCallable(() -> objMp.writeValueAsString(new AccountEmailContact(accountId, contact)))
-                             .getOrElseGet(Throwable::toString);
+                             .getOrElseThrow(JsonParseException::new);
             indexer.index(DOCUMENT_ID, content, ROUTING_KEY);
         });
```

Comment:
```
return Mono.fromCallable(() -> objMp.writeValueAsString(new AccountEmailContact(accountId, contact))
    .flatMap(content -> )indexer.index(DOCUMENT_ID, content, ROUTING_KEY);
```

Reply:
objMp -> objectMapper (we have auto-completion, let's make the code more readable by avoiding shortcuts...)